### ISSUE TITLE
improvement ideas for annex_prepare / annex / annex_stats

### ISSUE THREAD

**Issue by gabroko** [IEA-EBC-Annex86/annex#20]
- allow for multiple "Other" variables, e.g. Other1, Other2, etc.
- allow to include "variable addititonal information" in config file, e.g. to define "Other" variable directly there, and not manually afterwards in xlsx

**Comment by retostauffer** [comment]
# "Other" variables

I like the idea of allowing for `Other1` , `Other2` etc. to give room for custom information, and allow to set custom boundary checks in the `config` object.

### Implementation

I've tried one approach which turned out to be very tricky, but have a second idea. What we could do is the following:

1. We define `Other1`, `Other2`, ... up to `Other20` (@gabroko or how many you think are realistically needed).
2. This means I do not have to change the entire validation and all the checks.
3. We allow the user to specify `$lower`, `$upper` in the `config` if, and only if, the variable name is `Other*`.
4. I then need to figure out how to carry this information along as an attribute such that I can write it into the XLSX file (Definitions) table. This is used by `annex_validation`, no changes needed there _and_ the boundaries are stored in the XLSX stats file, so one can check what the statistics are based on.

# Units

We could do the same for `$unit` (for `Other*` variables only) and also write it into the stats XLSX file. What will not be possible is "automatic unit conversion" as this requires a specific set of rules (functions).

Would that work for you @gabroko ?

**Comment by gabroko** [comment]
Yes, that sounds perfect! I would think max 10 Other are enough. But no rush on that...
